web-app/js/portal/cart/PythonDownloadHandler.js

```diff
@@ -37,7 +37,7 @@ Portal.cart.PythonDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
         var _this = this;
 
         return function(collection) {
-            return collection.getLayerState().getSelectedLayer().getFeatureRequestUrl(
+            return OpenLayers.Layer.WMS.prototype.getFeatureRequestUrl(
```

Comment:
That's fair. I am happy to leave it here (with or without the *prototype* as needed). We can always move it later if we come up with a better place.